custom/conf/app.example.ini

```
@@ -2041,6 +2041,8 @@ PATH =
 ;ENABLED = false
 ;; If you want to add authorization, specify a token here
 ;TOKEN =
+;; Enable issue by label metrics; default is false
+;ENABLED_ISSUE_BY_LABEL = false
```

Please add an entry inside `docs/content/doc/advanced/config-cheat-sheet.en-us.md` as well.

---

Also, I think it would be more descriptive if this setting was called
```suggestion
;ENABLE_ISSUE_BY_LABEL_METRICS = false
```
That, however, is just personal preference. I don't know how you and/ or others think about that.
And, while I do see that there is duplication between `[metrics]` and the name, I still think it would be better in this specific case, as it is more clear this way what exactly this setting refers to.
Otherwise, one could wonder if `issue by label` is some kind of metrics generation technique.

---

I changed setting name and add a line in config-cheat-sheet file.